src/sentry/types/region.py

```diff
@@ -306,3 +306,12 @@ def find_all_multitenant_region_names() -> List[str]:
         for region in load_global_regions().regions
         if region.category == RegionCategory.MULTI_TENANT
     ]
+
+
+def get_region_display_name(region_name: str) -> str:
+    if region_name == "us":
+        return "🇺🇸 United States of America (US)"
+    if region_name == "de":
+        return "🇪🇺 European Union (EU)"
```

Comment:
Is there a better way of doing this check, and is this the appropriate place to put this method? Seems like something we'll need to potentially surface in multiple views/APIs.

Reply:
I think this location is fine. It is co-located with all the other region configuration related functions.  It might be better to make the display name an optional part of the `Region` type to co-locate it with other region data, and allow local environment region names to not stay in lockstep with region configuration defined in configuration management for saas.

Reply:
Agreed. I think this is something we can explore when we tackle ST integration as well since this will likely need to be propagated up from the system region config somehow.